app/models/idea.rb

```diff
@@ -1,3 +1,5 @@
 class Idea < ActiveRecord::Base
+  validates :title, format: { with: /\A[a-zA-Z0-9,?! ]+\z/ }
+  validates :body, format: { with: /\A[a-zA-Z0-9,?! ]+\z/ }
```

Comment:
@dastinnette, solid use of regex to deal with this issue.

One additional consideration....what if a user would like to use a `.` in the body of their idea?